packages/css/src/components/accordion/accordion.scss

```diff
@@ -0,0 +1,445 @@
+$dso-accordion-default-border-color: $cannon-pink;
+$dso-accordion-default-color: $cannon-pink;
+$dso-accordion-default-nested-handle-bgcolor: $cannon-pink;
+$dso-accordion-default-nested-body-bgcolor: #dbc8d1;
```

Comment:
Moeten we de kleur benoemen met een eigen kleurvariable?

Reply:
Ik ben doorgegaan op wat er ongeveer al stond. Persoonlijke keuze, denk ik? Zie je het liever anders?

Reply:
Ik vind niet dat je het kan afschuiven op persoonlijke keuze: we hebben immers `colors.scss`. En als ik de rest van het project bekijk vind ik dat we daar erg goed mee omgaan (afgezien van 'n paar `#ffffff` of `#efefef`).

> Ik ben doorgegaan op wat er ongeveer al stond.

Juist bij een refactor van `dso-toolkit` naar `css` is het belangrijk het fundament te herzien en niet "doorgaan op wat er stond".

Verder, als ik de specificaties van Bram bekijk valt het mij op dat de ingecheckte kleurcode (`#dbc8d1`) anders is dan wat hij hanteert (`#DCC8D2`). Misschien is het wel 'n lichtere kleur van `$cannon-pink` (`#8b4a6a`) en kunnen we er een [SASS kleurformule](https://sass-lang.com/documentation/modules/color) voor inzetten.

@bramhoosemans kan jij wat vertellen over de achtergrondkleur voor een accordion die geneste accordions bevat?

Reply:
Ik heb je vraag verkeerd geinterpreteerd, ik dacht dat je geen gebruik wilde maken van `$dso-accordion-default-nested-body-bgcolor` en de kleur direct in de code wilde zetten.

Ik ben het met je gedachte eens, dat hangt zeker niet af van persoonlijke voorkeur.

Reply:
> @bramhoosemans kan jij wat vertellen over de achtergrondkleur voor een accordion die geneste accordions bevat?


Ik ben er even ingedoken en kan zeggen de achtergrondkleur (`#DCC8D2` of `#dbc8d1`) niet te relateren zijn aan de `$cannon-pink`. Die we in ontwerp 'mauve' noemen trouwens. We hebben nog een taak op de backlog staan om de gebruikte kleuren beter te documenteren en ook technisch af te stemmen. 

In Sketch zijn we begonnen met een nieuwe gedeelde Library waarin het kleurgebruik ook beter in variabelen gezet zijn. In dit geval zouden we al de gewijzigde waarde kunnen implementeren (mogelijk met SASS kleurformule?).

De achtergrondkleur is `#D1B7C4` (40% `$cannon-pink` `#8B4A6A` op wit).

![Schermafbeelding 2021-12-21 om 10 13 14](https://user-images.githubusercontent.com/598735/146903742-2c6352dd-cc22-4ecd-8ae2-36699c7192da.png)

Reply:
Plot twist:

![image](https://user-images.githubusercontent.com/9696261/147076613-700c5b1a-b03e-4cdd-829c-0cf099df71cf.png)